version/version.go

```diff
@@ -1,5 +1,21 @@
 // Package version provides shared version information.
 package version
 
+import (
+	"fmt"
+)
+
 // Version is the current version of MicroCloud.
-const Version = "1.1"
+const Version = "2.1"
+
+// LTS should be set if the current version is an LTS (long-term support) version.
+const LTS = true
+
+// LTSVersion appends "LTS" to the version string if MicroCloud is an LTS version.
+func LTSVersion() string {
```

Comment:
Why not just Version()? Seems odd to have a function called LTSVersion not always return the LTS suffix.

Reply:
I've now renamed the raw version const to `RawVersion` and this helper is just `Version`.